moviesite/polls/views.py

```diff
@@ -0,0 +1,10 @@
+from django.http import HttpResponse
+import movie_api
+
+def hello(request):
+    return HttpResponse("Hello, world. You're at the polls index.")
+
+def index(request,movie):
+    str=movie_api.get_data(movie)
```

Comment:
`str` is a reserved keyword in python. Your application could break in weirdest ways possible if you get into that habit